src/Core/Entities/Test.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Realworlddotnet.Core.Entities;
+
+public class Test
+{
+    [Key]
+    public Guid Id { get; set; }
+
+    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
+
+    public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
```

Comment:
Consider adding a method to update the UpdatedAt property automatically whenever the entity is modified. This ensures that the timestamp is always accurate and reduces the risk of human error.